packages/chart/src/CdcChart.tsx

```diff
@@ -385,11 +385,9 @@ const CdcChart: React.FC<CdcChartProps> = ({
         if (configObj.data && configObj) {
           const preparedConfig = await prepareConfig(configObj, data)
           const preparedData = prepareData(configObj, data)
-
           setStateData(preparedData)
           setExcludedData(preparedData)
           updateConfig(preparedConfig, preparedData)
-          setFilteredData(filterVizData(config.filters, data))
```

Comment:
I saw an issue here because you were referencing config instead of configObj. maybe you don't need to remove this entirely